public/js/tests/tests.js

```diff
@@ -1,90 +1,79 @@
-test("should sort channels by name", function(assert) {
-  var channels = [
-    {friendlyName: "BBB"},
-    {friendlyName: "BBA"}
-  ];
+document.body.innerHTML = window.__html__['views/index.jade'];
 
-  var result = twiliochat.sortChannelsByName(channels)
+describe('TwilioChat', function() {
+  it('should sort channels by name', function() {
+    var channels = [
+      {friendlyName: 'BBB'},
+      {friendlyName: 'BBA'}
+    ];
 
-  assert.deepEqual(result, [{friendlyName: "BBA"}, {friendlyName: "BBB"}]);
-});
-
-test("should be able to sort an empty list of channels", function(assert) {
-  var channels = [];
+    var result = twiliochat.sortChannelsByName(channels)
 
-  var result = twiliochat.sortChannelsByName(channels)
+    assert.deepEqual(result, [{friendlyName: 'BBA'}, {friendlyName: 'BBB'}]);
+  });
 
-  assert.deepEqual(result, []);
-});
+  it('should be able to sort an empty list of channels', function() {
```

Comment:
This example makes me think we're missing a context - looks like we're trying to describe the function `sortChannelsByName`.

```javascript
describe("sortChannelsbyName", function() {
  it("sorts channels by name", function() {
    // ...
  });

  it("sorts an empty list", function() {
    // This case is good, since an empty list
    // is the obvious edge case for a list.
  });
});